### ISSUE TITLE
guidelines: encoding errors

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/music-encoding#1299]
* `01-introduction.xml`
The last two paragraphs in `aboutVersion` are not allowed at this point.
* `05-mensural.xml`
In `mensuralSigns` attribute values are encoded within `att`, which is not allowed. Also the id (`mensuralSigns`) differ from the name (`Mensuration`)
* `08-lyricsperfdir.xml` and `09-textencoding.xml`
`respStmt` is missing an editor name. At least an empty `name` element should be added.

**Comment by musicEnfanthen** [comment]
The error about the last two paragraphs in `01-introduction.xml` is mysterious.

The structure is:

```html
<div>
  <head>...</head>
  <p>...</p>
  <div>---</div>
  <div>---</div>
  <div>---</div>
  <div>---</div>
  <p>...</p> <!-- error about these lines: "element 'p' not allowed here"
  <p>...</p>
</div>
```

Any ideas why this shouldn`t be allowed?

**Comment by rettinghaus** [comment]
TEI is using this line for their guidelines: 

`<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>`

Seems to work fine with this. We could just switch for the Guidelines …
(Though I'm not really sure how to validate with this.)

**Comment by pe-ro** [comment]
This has to do with what was once called "tesselation".  `<p>` can't follow `<div>` in with a `<div>`. The solution is to wrap the last 2 paragraphs in `<div>`, e.g.,
```
<div>
  <head>...</head>
  <p>...</p>
  <div>---</div>
  <div>---</div>
  <div>---</div>
  <div>---</div>
  <div><p>...</p> <!-- error about these lines: "element 'p' not allowed here"
  <p>...</p></div>
</div>
```

**Comment by bwbohl** [comment]
As the content model of `tei:div` shows it has as content (amongst other sequences) either a sequence of divLike elements or  model.global (which contains model.divPart which contains model.pLike.

**Comment by musicEnfanthen** [comment]
But you can have a `<p>` before all other inner `<divs>`... nice.

Thank you all for clarification. Will embrace those p's with another div.

**Comment by musicEnfanthen** [comment]
As it turns out, it is a bit more complicated than the suggested fix. The wrapping div is not allowed to have no attributes, but needs an id and level type (otherwise the XSLT for building the guidelines complains about an empty element). As soon as you add those, the PDF guidelines will create another "empty" subchapter.... 👀 

Workaround for now: I added those two paragraphs to the end of the previous div.

**Comment by bwbohl** [comment]
can this be closed since #1301 was merged?

**Comment by rettinghaus** [comment]
Yes
